src/com/biocollapse/controller/SimulationController.java

```diff
@@ -62,9 +63,15 @@ public void runMainLoop() {
 
                 double fps = lastFps; // Needed for swingutilities to access scope.
                 SwingUtilities.invokeLater(() -> {
-                    // TODO: Get actual statistics.
                     LiveStatistics newLiveStatistics = simulationService.calculateLiveStatistics(humans, hospitals);
-                    visualisation.update(humans, newLiveStatistics, fps);
+
+                    // For testing, please set testGraphs to true. TODO: Remove testing code later.
+                    boolean testGraphs = false;
+                    if (testGraphs) {
```

Comment:
Please remove the test code before merging the pr. 😉

Reply:
Done! Thanks for feedback. Glad it works now.